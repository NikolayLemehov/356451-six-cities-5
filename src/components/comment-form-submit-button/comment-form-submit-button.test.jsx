import React from "react";
import renderer from "react-test-renderer";
import CommentFormSubmitButton from "./comment-form-submit-button";

describe(`CommentFormSubmitButton should render correctly`, () => {
  it(`CommentFormSubmitButton should render correctly when isDisabledSubmitButton={false}`, () => {
    const tree = renderer
      .create(<CommentFormSubmitButton
        isDisabledSubmitButton={false}
      />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
  it(`CommentFormSubmitButton should render correctly when isDisabledSubmitButton={true}`, () => {
    const tree = renderer
      .create(<CommentFormSubmitButton
        isDisabledSubmitButton={true}
      />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
