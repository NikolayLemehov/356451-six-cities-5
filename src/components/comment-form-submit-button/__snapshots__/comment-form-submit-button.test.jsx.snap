// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CommentFormSubmitButton should render correctly CommentFormSubmitButton should render correctly when isDisabledSubmitButton={false} 1`] = `
<button
  className="reviews__submit form__submit button"
  disabled="disabled"
  type="submit"
>
  Submit
</button>
`;

exports[`CommentFormSubmitButton should render correctly CommentFormSubmitButton should render correctly when isDisabledSubmitButton={true} 1`] = `
<button
  className="reviews__submit form__submit button"
  type="submit"
>
  Submit
</button>
`;
